Q9/FirstMissingInt.rb

```diff
@@ -0,0 +1,26 @@
+def missingint(array)
+    array = []
+    index = 0
+    array.each do |x|
+     binary_search(array, array[index])
+     index += 1
```

Comment:
How does it return the first integer number is missing? Looks like index will always equal the size of the array and  not the first missing integer :/ I might not be understanding right tho lmk!